models/room.go

```diff
@@ -557,12 +558,19 @@ func GetRoomDetails(redisClient interfaces.RedisClient, schedulerName string, ro
 
 	lastPingAt, _ := strconv.ParseInt(roomData["lastPing"], 10, 64)
 	status := roomData["status"]
+	var lastPingMetadata map[string]interface{}
```

Comment:
@lyracampos  i think we can add a test case here for when the "metadata" key does not exist on roomData, this is a scenario that will happen when the incoming ping does not have metadata when persisting it (you can check line 279 in this file). For this scenario, I don't think this function should return an error in this scenario, it would be best for it to return empty metadata, WDYT?.